module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -134,7 +135,24 @@ public function hasCatalog()
     public function pubTypeRtacEnabled()
     {
         $pubTypeId = $this->fields['Header']['PubTypeId'];
-        return $pubTypeId !== 'ebook';
+        return !($pubTypeId === 'ebook');
+    }
+
+    /**
+     * Return the relevancy score for this record.
+     *
+     * @return ?float
+     */
+    public function getScore()
```

Comment:
Given that the two Results functions for accessing scores are initially defined in the base class, we maybe should do the same for the RecordDriver and add `getScore` to the method master list.  But I don't know how much it would buy us.

Reply:
I think the potential problem with making `getScore` a universal method across record drivers is that doing so could imply that the scores are somehow equivalent across backends, but they never will be. Consider, for example, what might happen if you tried to use a score-based recommendation module placement in a Blender-driven result set; I suspect that would be a very bad idea.

That's not to say that I don't think you should have `getScore` methods in multiple record drivers -- just that I think we need to be careful not to imply that these scores should be used in ways that in reality will not be meaningful or helpful.